### PR TITLE
Added table containing metadata items

### PR DESCRIPTION
**Pull request by Alopalao**
Based on issue #54.
It have been added a table with the metadata items and active, enabled keys to basic details. What would be the triggers for the notifications?

### CODE REVIEW

File: ui/k-info-panel/switch_info.kytos [hunk 1]
```diff
@@ -36,7 +55,9 @@
                   'manufacturer': '',
                   'serial': '',
                   'hardware': '',
-                  'software': ''},
+                  'software': '',
+                  'enabled': '',
+                  'active': '',},
```
Comment (viniarck) [inline ui/k-info-panel/switch_info.kytos:60]: @Alopalao let's switch the order of both `enabled` and `active` to come before the `id`, that way in the property panel it's always going to start with `enabled` and `active` followed by the rest of the properties in the same way that you've done for the link details on this PR https://github.com/kytos-ng/topology/pull/73/files. As it is, depending on the screen size it'd be at the bottom requiring scrolling, and this is a very important info that operators will look for:

2022-03-10-171508_417x410_scrot

**Comment by viniarck** [comment]
> Based on issue #54. It have been added a table with the metadata items and active, enabled keys to basic details. What would be the triggers for the notifications?

@Alopalao, please make sure to also update the changelog. Regarding the triggers for notifications, no need for them in this case since you're leveraging the props content that is being passed.

**Comment by Alopalao** [comment]
Changelog.rst is updated. Commit [8e67f19a9d9fd4a804394fe83cfec068dbc1f9cc](https://github.com/kytos-ng/topology/pull/75/commits/8e67f19a9d9fd4a804394fe83cfec068dbc1f9cc)

File: ui/k-info-panel/switch_info.kytos (hunk 1 above)
Comment (Alopalao) [reply ui/k-info-panel/switch_info.kytos]: The order of the key has ben changed on the commit  [8e67f19a9d9fd4a804394fe83cfec068dbc1f9cc](https://github.com/kytos-ng/topology/pull/75/commits/8e67f19a9d9fd4a804394fe83cfec068dbc1f9cc)

File: ui/k-info-panel/switch_info.kytos [hunk 2]
```diff
@@ -76,3 +102,37 @@
...
+<style>
+.metadata_table {
+  color: #ccc;
+  max-height: 250px;
+  text-align: center;
+  margin: 0 auto;
+  display: block;
+  padding: 0.5em 0 1em 0.3em;
+  font-size: 0.8em;
+  height: 190px;
```
Comment (rmotitsuki) [inline ui/k-info-panel/switch_info.kytos:114]: Need an overflow property for table with static height value, for the cases with too many data. You can test adding 20 more fake metadata_items to check how it is rendered.

**Review by rmotitsuki — CHANGES_REQUESTED** [review]

File: ui/k-info-panel/switch_info.kytos (hunk 2 above)
Comment (Alopalao) [reply ui/k-info-panel/switch_info.kytos]: Now it handles overflow. Commit [36ffa8d66af7004f9e4d43f478c5f798ab9da12b](https://github.com/kytos-ng/topology/pull/75/commits/36ffa8d66af7004f9e4d43f478c5f798ab9da12b)

**Comment by Alopalao** [comment]
The table when overflows looks like this:
image
Is it okay? I was wondering if it would be better to have a scroll bar for x-axis when the cell overflows like in link_name. Right now the first column is fixed to 62px.

**Comment by rmotitsuki** [comment]
> as 62px I think it's too small based on what we expect to use in production
I don't have any great suggestions here... Just think of the worst case scenario for production and increase the width accordingly. Maybe increase to 100px will be enough for most cases.

**Comment by Alopalao** [comment]
Decided to remove the fixed width. Commit [5f70e0d23565e50da4e97d62f9f7139a2d26dc29](https://github.com/kytos-ng/topology/pull/75/commits/5f70e0d23565e50da4e97d62f9f7139a2d26dc29)

**Review by rmotitsuki — APPROVED** [review]
Looks good to me.

File: ui/k-info-panel/switch_info.kytos [hunk 3]
```diff
@@ -259,6 +283,39 @@
...
+  vertical-align: middle;
+  padding: 0.45em 0 0.45em 0;
+  word-break: break-all;
+}
+.metadata_table tbody tr:nth-child(even) {
+  background: #313131;
+}
+.metadata_table tbody tr:hover {
+    color: #eee;
+    background-color: #666;
```
Comment (viniarck) [inline ui/k-info-panel/switch_info.kytos:318]:
```suggestion
    background-color: #666;
    }
```

**Review by viniarck — APPROVED** [review]

**Review by viniarck — APPROVED** [review]
Well done @Alopalao, awesome to have this UI functionality, it'll be very useful for network operators. I've just solved the git conflicts, I'll merge this PR.

**Pull request merged**